docs/src/test-projects-js.md

```diff
@@ -206,6 +205,61 @@ Running order:
 
 <img width="70%" style={{display: 'flex', margin: 'auto'}} alt="Browser login project is blue, database is red and e2e tests relies on both" loading="lazy" src="https://user-images.githubusercontent.com/13063165/225938262-33c1b78f-f092-4762-a478-7f8cbc1e3b21.jpg" />
 
+### Teardown
```

Comment:
I've just read this in `docs/src/test-global-setup-teardown-js.md` - is this supposed to be this way?

Reply:
I would do a reference instead of copying the same text in two places